docs/reference_guide.md

```diff
@@ -9,137 +9,6 @@ Also, please keep these docs as terse as possible to maintain their brevity (ins
-    - [15. `kstack()`: Stack Traces, Kernel](#15-kstack-stack-traces-kernel)
-    - [16. `ustack()`: Stack Traces, User](#16-ustack-stack-traces-user)
-    - [17. `cat()`: Print file content](#17-cat-print-file-content)
-    - [18. `signal()`: Send a signal to the current task](#18-signal-send-a-signal-to-current-task)
-    - [19. `strncmp()`: Compare first n characters of two strings](#19-strncmp-compare-first-n-characters-of-two-strings)
-    - [20. `strcontains()`: Compares whether the string haystack contains the string needle.](#20-strcontains-compare-whether-the-string-haystack-contains-the-string-needle)
-    - [21. `override()`: Override return value](#21-override-override-return-value)
-    - [22. `buf()`: Buffers](#22-buf-buffers)
-    - [23. `sizeof()`: Size of type or expression](#23-sizeof-size-of-type-or-expression)
-    - [24. `print()`: Print Value](#24-print-print-value)
-    - [25. `strftime()`: Formatted timestamp](#25-strftime-formatted-timestamp)
-    - [26. `path()`: Return full path](#26-path-return-full-path)
-    - [27. `uptr()`: Annotate userspace pointer](#27-uptr-annotate-userspace-pointer)
-    - [28. `kptr()`: Annotate kernelspace pointer](#28-kptr-annotate-kernelspace-pointer)
-    - [29. `macaddr()`: Convert MAC address data to text](#29-macaddr-convert-mac-address-data-to-text)
-    - [30. `cgroup_path()`: Convert cgroup id to cgroup path](#30-cgroup_path-convert-cgroup-id-to-cgroup-path)
-    - [31. `bswap`: Reverse byte order](#31-bswap-reverse-byte-order)
-    - [32. `skb_output`: Write `skb` 's data section into a PCAP file](#32-skb_output-write-skb-s-data-section-into-a-pcap-file)
-    - [33. `pton()`: Convert text IP address to byte array](#33-pton-convert-text-ip-address-to-byte-array)
-    - [34. `strerror`: Get error message for errno value](#34-strerror-get-error-message-for-errno-code)
-    - [35. `offsetof`: Offset of element in structure](#35-offsetof-offset-of-element-in-structure)
-    - [36. `nsecs()`: Timestamps and Time Deltas](#36-nsecs-timestamps-and-time-deltas)
-- [Map Functions](#map-functions)
-    - [1. Builtins](#1-builtins-2)
-    - [2. `count()`: Count](#2-count-count)
-    - [3. `sum()`: Sum](#3-sum-sum)
-    - [4. `avg()`: Average](#4-avg-average)
-    - [5. `min()`: Minimum](#5-min-minimum)
-    - [6. `max()`: Maximum](#6-max-maximum)
-    - [7. `stats()`: Stats](#7-stats-stats)
-    - [8. `hist()`: Log2 Histogram](#8-hist-log2-histogram)
-    - [9. `lhist()`: Linear Histogram](#9-lhist-linear-histogram)
-    - [10. `print()`: Print Map](#10-print-print-map)
-- [Output](#output)
-    - [1. `printf()`: Per-Event Output](#1-printf-per-event-output)
-    - [2. `interval`: Interval Output](#2-interval-interval-output)
-    - [3. `hist()`, `printf()`: Histogram Printing](#3-hist-print-histogram-printing)
-    - [4. `SIGUSR1`: On-Demand Output](#4-sigusr1-on-demand-output)
-- [BTF Support](#btf-support)
-- [Advanced Tools](#advanced-tools)
-- [Errors](#errors)
```

Comment:
I'm a bit confused about the deleted entries here. It doesn't look like it was moved into the man page. Is that intentional?

Reply:
Yes, this is intentional.

I'm moving contents those should be included in the man page, and deleting those should not. So when the reference guide become empty, the man page is complete.

Reply:
Ah ok, I see. Your intention is to "delete" the table of contents in this PR. But the actual contents will be moved in subsequent PRs.